code/modules/clothing/head/helmet.dm

```diff
@@ -203,19 +194,27 @@ obj/item/clothing/head/blob
 	flags = HEADCOVERSEYES|HEADCOVERSMOUTH
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES
 
-/obj/item/clothing/head/helmet/knight/blue
+/obj/item/clothing/head/helmet/riot/knight

```

Comment:
Might want to move this, and the rest of the knight helmets to just below the riot helmet, that way it's easier to immediately see what gets inherited.